### DISCUSSION TITLE
Arm support

### DISCUSSION THREAD

**Discussion by wolfianer** [JaidedAI/EasyOCR#1163] [Q&A]
I find nothing on the compatibility on arm.
Has anyone ever tried to run this on arm64 on for instance a raspberry pi? 
Installation works flawless, but i get a segmentation fault during the construction of the easyocr.Reader()

**Comment by pflajszer** [comment]
+1 from me. I'm having issues building a docker container with easyocr dependency, which initially was developed using linux/amd64 arch.

**Reply by Potracheno** [reply]
Have you succeeded ?
